tests/testthat/test-function_argument_linter.R

```diff
@@ -0,0 +1,9 @@
+test_that("function_argument_linter returns the correct linting", {
```

Comment:
add some tests of multi-line signatures. XPath means we _should_ be fine, but no harm to include.

the same logic applies to lambda functions too, right? I think typically people are not defining lambdas with defaults, but I think syntactically it's OK (don't have R 4.1 in front of me ATM) to write `\(x, y = 2, z)`. Are those in scope?

Reply:
Added them. Just requires `//OP-LAMBDA` in addition to `//FUNCTION`.